nuxeo-core/nuxeo-core-test/src/main/java/org/nuxeo/ecm/core/test/CoreFeature.java

```diff
@@ -165,11 +163,14 @@ public void start(FeaturesRunner runner) {
         try {
             RuntimeHarness harness = runner.getFeature(RuntimeFeature.class).getHarness();
             storageConfiguration.init();
+            for (String bundle : storageConfiguration.getExternalBundles()) {
+                harness.deployBundle(bundle);
+            }
             URL blobContribUrl = storageConfiguration.getBlobManagerContrib(runner);
             harness.getContext().deploy(new URLStreamRef(blobContribUrl));
             URL repoContribUrl = storageConfiguration.getRepositoryContrib(runner);
             harness.getContext().deploy(new URLStreamRef(repoContribUrl));
-        } catch (IOException e) {
+        } catch (Exception e) {
```

Comment:
We don't like to catch `Exception`, so if you really have to (due to stupid `deployBundle` signature) do it right around the incriminated method.